### DISCUSSION TITLE
Ability to see or group by package manager type in Dependency Dashboard

### DISCUSSION THREAD

**Discussion by mscrivo** [renovatebot/renovate#18016] [Suggest an Idea]
### What would you like Renovate to be able to do?

We can already apply labels per package manager type so we can sort already created PRs by those labels, however, in the dependency dashboard, it's difficult to figure out which items belong to which package manager type. For example, you could have two similarly named packages from two different package manager systems and they would show up as identical line items in the dependency dashboard. It would be nice to know which package manager type each line referred to.

### If you have any ideas on how this should be implemented, please tell us here.

My initial ideas would be to allow either:
1. the package manager type to be appended to each line. Something like:

- [ ] [npm] Update dependency async to v3
- [ ] [bundler] Update dependency async to v3

2. grouping the line items by package manager type. 

### Is this a feature you are interested in implementing yourself?

Maybe

**Comment by rarkins** [comment]
PRs can group dependencies from many managers

**Comment by mscrivo** [comment]
> PRs can group dependencies from many managers

ah I did not know that .. haven't seen it ever do that.

**Comment by rarkins** [comment]
Hi, I have converted this from feature request to discussion due to the lack of clear direction plus that I think you can achieve most of what you want with config.

The Dashboard reflects the PRs titles, so if you can get the package manager into the PR title, then you'll see it in dashboard. PR titles default to copying the commitMessage, and commitMessage has the concept of `commitMessageSuffix`.

Check out this example: https://github.com/renovate-reproductions/dashboard-titles/issues/4

![image](https://user-images.githubusercontent.com/6311784/192713744-778919bd-4cc5-4735-bbf7-7041435b1da2.png)
